pkg/operator/status.go

```diff
@@ -385,6 +386,123 @@ func (optr *Operator) syncMetrics() error {
 	return nil
 }
 
+func (optr *Operator) syncClusterFleetEvaluation() error {
+	co, err := optr.fetchClusterOperator()
+	if err != nil {
+		return err
+	}
+	if co == nil {
+		return nil
+	}
+
+	unexpectedEvaluations, err := optr.generateClusterFleetEvaluations()
+	if err != nil {
+		return err
+	}
+
+	var coStatus configv1.ClusterOperatorStatusCondition
+	if len(unexpectedEvaluations) > 0 {
+		coStatus = configv1.ClusterOperatorStatusCondition{
+			Type:   configv1.EvaluationConditionsDetected,
+			Status: configv1.ConditionTrue,
+			Reason: evaluationToString(unexpectedEvaluations),
+		}
+	} else {
+		coStatus = configv1.ClusterOperatorStatusCondition{
+			Type:   configv1.EvaluationConditionsDetected,
+			Status: configv1.ConditionFalse,
+			Reason: asExpectedReason,
+		}
+	}
+
+	return optr.updateStatus(co, coStatus)
+}
+
+func evaluationToString(e []string) string {
+	if len(e) == 0 {
+		return ""
+	}
+	return strings.Join(e, "::")
+}
+
+func (optr *Operator) generateClusterFleetEvaluations() ([]string, error) {
+	evaluations := []string{}
+
+	enabled, err := optr.cfeEvalFailSwapOn()
+	if err != nil {
+		return evaluations, err
+	}
+	if enabled {
+		evaluations = append(evaluations, "failswapon")
+	}
+
+	enabled, err = optr.cfeEvalCrun()
+	if err != nil {
+		return evaluations, err
+	}
+	if enabled {
+		evaluations = append(evaluations, "crun")
```

Comment:
I think we can assume if the flag is not set then it's whatever default we have.

Reply:
I already added the cgroupv1 case.

Reply:
sorry as in adding runc/cgroupv2 . I do wonder if having all cases for the runtime will be useful in case we add others in the future (like a conmon-rs runtime class)